arch/x86_64/src/intel64/Kconfig

```diff
@@ -68,6 +68,10 @@ config ARCH_INTEL64_TSC
 	---help---
 		Select to enable the use of TSC APIC timer of x86_64
 
+config ARCH_INTEL64_HPET_ALARM
+	bool "HPET timer alarm support"
+	depends on ALARM_ARCH
```

Comment:
Please include --help-- with some useful information to users, when to use HPET Alarm instead other timer, etc